### DISCUSSION TITLE
SSL Certificate Error

### DISCUSSION THREAD

**Discussion by Curescg** [community/community#110480] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My Site was working fine yesterday with the certificate and Https: today it shows github.io certificate
![WhatsApp Image 2024-02-29 at 11 38 04 AM](https://github.com/community/community/assets/161235521/1d9a3214-325c-4d6c-b2bf-df7274ecac6f)
<img width="1247" alt="Screenshot 2024-02-29 at 11 50 09 AM" src="https://github.com/community/community/assets/161235521/06e379f9-e970-4222-994b-6149ae119bf1">

**Comment by ZuhairTarif** [comment]
Use cloudflare for free ssl, in this way you will face problem

**Reply by Curescg** [reply]
so google ssl causes an 
issue?

**Reply by ZuhairTarif** [reply, marked as answer]
yeah sometimes, that's why I always use Cloudflare. No problem found till now and it's so easy + free

**Reply by Curescg** [reply]
Thanks will try it out.
